block-library/freeform/editor.scss

```diff
@@ -115,6 +115,20 @@
 }
 
 .editor-block-list__layout .editor-block-list__block[data-type="core/freeform"] {
+
+	// Not sure why this is necessary, there seems to be a skin file that overrides this upstream.
```

Comment:
This is from a skin file, it looks like this without it:

<img width="541" alt="screen shot 2018-09-07 at 10 28 24" src="https://user-images.githubusercontent.com/1204802/45207640-c5491e80-b288-11e8-99d2-f84c90f67955.png">

It's from /wp-includes/js/tinymce/skins/lightgray/skin.min.css

```
.mce-btn.mce-active button, .mce-btn.mce-active:hover button, .mce-btn.mce-active i, .mce-btn.mce-active:hover i {
    color: white;
}
```

Should we be overriding or can we de-queue this one?